workshops/management/commands/check_for_workshop_websites_updates.py

```diff
@@ -97,6 +99,11 @@ def get_events(self):
         This method is used for getting all events that should be checked
         against up-to-date data."""
         events = Event.objects.active().filter(url__isnull=False)
+
+        # events as old as 2014 are still marked as active, so we impose age
+        # limit of half a year
+        half_a_year = datetime.timedelta(days=182)
```

Comment:
Yes. Cuts off plenty of old events who still aren't marked as "completed" even though it's been few years.